scripts/annotation.js

```diff
@@ -11,10 +11,34 @@ $(document).ready( ()=> {
     $("#structure").text($.urlParam('structure'));
     index = $.urlParam('index');
     $("#image").attr('src', '/images?index=' + index);
+  
+    document.onkeyup = function(event) {
+        var e = (!event) ? window.event : event;
+        switch (e.keyCode) {
+            //left arrowkey
+            case 37:
+                prevImage();
+                break;
+            //right arrowkey
+            case 39:
+                nextImage();
+                break;
+        }
+    };
 
     $("#image").click( ()=> {
-        index++;
-        if (index > 9) index = 0;
-        $("#image").attr('src', '/images?index=' + index);
+        nextImage();
```

Comment:
Separating this is a good call, because then when we want left and right to be annotations we just swap out the function.